skipped_tests.toml

```diff
@@ -93,6 +93,7 @@ unit = [
 ]
 integration = [
 	"tests/integration/cli/test_custom_module.py::SSHCustomModuleTest::test_ssh_custom_module",  # Needs investigation: Fails only with Salt Bundle
+	"tests/integration/cli/test_custom_module.py::SSHCustomModuleTest::test_ssh_regular_module", # Needs investigation: Fails only with Salt Bundle
```

Comment:
Where does this fail? We're getting green on Salt Bundle pipelines (with obvious exceptions, e.g. Debian 11 or Ubuntu 24.04).